ddp-workspace/projects/ddp-dsm-ui/src/app/participant-list/participant-list.component.ts

```diff
@@ -1020,11 +1020,14 @@ export class ParticipantListComponent implements OnInit {
               }
             } else {
               // if selected columns are not set, set to default columns
+              const selectedStudy = localStorage.getItem(ComponentService.MENU_SELECTED_REALM);
               if ((this.selectedColumns['data'] && this.selectedColumns['data'].length === 0)
                 || (!this.selectedColumns['data'] && this.isSelectedColumnsNotEmpty())) {
-                this.dataSources.forEach((value: string, key: string) => {
-                  this.selectedColumns[key] = [];
-                });
+                if(selectedStudy !== 'RGP') {
```

Comment:
I am not aware much about the functionality.
But that fix seems like a hack.
The question is why **RGP** behaves differently in comparison with other activities?
In my opinion the bug cause is something else.

Reply:
I have written why it behaves differently in the description. I am still working on this PR. I spotted another bug caused by this solution